src/ClaimService.Models.Db/DbClaimFile.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace LT.DigitalOffice.ClaimService.Models.Db;
+
+public class DbClaimFile
+{
+  public const string TableName = "ClaimFiles";
+
+  public Guid Id { get; set; }
+  public Guid ClaimId { get; set; }
+  public Guid FileId { get; set; }
```

Comment:
Same. If you don't configure it properly, you won't be able to use nested entities by getting them from _provider.Include()